### ISSUE TITLE
Error during widget testing. (An animation is still running even after the widget tree was disposed)

### ISSUE THREAD

**Issue by Flexonze** [2d-inc/Nima-Flutter#12]
Hi, I am getting this error while running a simple widget test: 

The following message was thrown:
An animation is still running even after the widget tree was disposed.
...

This is the test:
```
testWidgets('DetailsPage displays properly', (WidgetTester tester) async {
    await tester.pumpWidget(
        Foo bar = FooFactory().create();
            MaterialApp(
              home: DetailsPage(foo: bar),
            ),
        );

    expect(find.byKey(Key('DetailsScaffoldKey')), findsOneWidget);
});
```
Any idea how i can fix this?

**Comment by Flexonze** [comment]
This is my Nima actor:

```
NimaActor(
    "Assets/Images/icon-thumb.nima",
    alignment: Alignment.center,
    animation: "spinning",
),
```

**Comment by Flexonze** [comment]
This issue was fixed in 1.0.3. Thanks to @luigi-rosso
